javasphinx/compiler.py

```diff
@@ -339,4 +339,9 @@ def compile(self, ast):
             document = self.compile_type_document(import_block, package, name, declaration)
             documents[full_name] = (package, name, document.build())
 
+
         return documents
+
+    def compile_docblock(self, documented):
+        ''' Compiles a single, standalone docblock. '''
+        return self.__output_doc(documented).build()
```

Comment:
Could you add a newline to the end of this file and remove the blank lines added above?